grpc-server-runtime/src/main/resources/META-INF/native-image/io.micronaut.grpc/micronaut-grpc-runtime/native-image.properties

```diff
@@ -0,0 +1 @@
+Args = --initialize-at-run-time=com.google.protobuf.ExtensionRegistryFactory
```

Comment:
Directory needs rename to `micronaut-grpc-server-runtime` or whatever this is appropriate for

Reply:
done!